openjdk/build.xml

```diff
@@ -26,57 +26,7 @@
 	<property environment="env" />
 	<property name="LIB" value="tohandler_simple"/>
 	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml"/>
-
-	<target name="getJtreg">
-		<mkdir dir="${DEST}"/>
-		<if>
-			<or>
-				<matches pattern="^[89]{1}$" string="${JDK_VERSION}"/>
-				<matches pattern="^[1][01234]" string="${JDK_VERSION}"/>
-			</or>
-			<then>
-				<property name="jtregTar" value="jtreg-4.2.0-tip.tar"/>
-			</then>
-			<else>
-				<property name="jtregTar" value="jtreg-5.1-b01.tar"/>
-			</else>
-		</if>
-		<if>
-			<not>
-				<available file="${jtregTar}" />
-			</not>
-			<then>
-				<if> <available file="custom_jtreg.tar.gz"/>
-					<then>
-						<echo message="Using custom_jtreg.tar.gz"/>
-							<copy file="custom_jtreg.tar.gz" tofile="${jtregTar}.gz"/>
-					</then>
-				<else>
-					<echo message="Getting Jtreg from https://ci.adoptopenjdk.net/job/dependency_pipeline/lastSuccessfulBuild/artifact/jtreg/${jtregTar}.gz"/>
-					<exec executable="curl" failonerror="true">
-						<arg line="--retry 5 -OLJSks https://ci.adoptopenjdk.net/job/dependency_pipeline/lastSuccessfulBuild/artifact/jtreg/${jtregTar}.gz" />
-					</exec>
-				</else>
-				</if>
-				<exec executable="gzip" failonerror="true">
-					<arg line="-d ${jtregTar}.gz" />				</exec>
-			</then>
-		</if>
-		<if>
-			<contains string="${SPEC}" substring="zos" />
-			<then>
-				<exec executable="tar" failonerror="true">
-					<arg line="xfo ${jtregTar} -C ${DEST}" />
-				</exec>
-			</then>
-		<else>
-			<exec executable="sh" failonerror="true">
-				<arg line="-c 'cat ${jtregTar} | (cd ${DEST} &amp;&amp; tar xof -)'" />
-
-			</exec>
-		</else>
-		</if>
-	</target>
+	
```

Comment:
We are downloading jtreg jar in https://github.com/adoptium/TKG/pull/189. In this PR, we need to set `LIB` based JDK_VERSION and JDK_IMPL. It can take comma-separated values. `LIB` tells getDependencies script which jar to download. We predefined `LIB` before import to override the default value.

Steps:
1. define `jtregjar` name based on JDK_VERSION (refer line 32-43). `jtregjar` name needs to match with https://github.com/adoptium/TKG/pull/189
2. set tohandler_simple if JDK_IMPL is ibm/openj9 (refer line 209-214)
3. set `LIB` properly
4. remove downloading logic in `getJtreg`. We need to keep unzip logic.

Reply:
Updated the build.xml file according to the steps defined. Please review the updates

Reply:
No, we need to add values in `LIB`
In summary, 

- please remove line27 and add the following code in line27:

```
<if>
  <or>
    <matches pattern="^[89]{1}$" string="${JDK_VERSION}"/>
    <matches pattern="^[1][01234]" string="${JDK_VERSION}"/>
  </or>
  <then>
    <property name="jtregTar" value="jtreg_4_2_0_tip"/>
	</then>
	<else>
    <property name="jtregTar" value="jtreg_5_1_b01"/>
	</else>
</if>
<if>
  <or>
    <equals arg1="${JDK_IMPL}" arg2="ibm"  />
    <equals arg1="${JDK_IMPL}" arg2="openj9" />
	</or>
	<then>
            <property name="openj9jtregtimeouthandler" value=",tohandler_simple"/>
	</then>
	<else>
         <property name="openj9jtregtimeouthandler" value=""/>
       </else>
</if>
<property name="LIB" value="${jtregTar}${openj9jtregtimeouthandler}"/>
```

- delete getDependentLibsCheck target https://github.com/adoptium/aqa-tests/blob/1e731fdb49c981baab26e18420be121aef6836e5/openjdk/build.xml#L262

- change getDependentLibsCheck to getDependentLibs https://github.com/adoptium/aqa-tests/blob/1e731fdb49c981baab26e18420be121aef6836e5/openjdk/build.xml#L274